lib/registries/addon/discover/template.hbs

```diff
@@ -0,0 +1,108 @@
+{{page-title (t 'registries.discover.page_title') prepend=false}}
+
+<RegistriesWrapper
+    @provider={{this.providerModel}}
+    {{with-branding (get-model this.providerModel.brand)}}
+    data-analytics-scope='Registries Discover page'
+>
+    <HeroOverlay local-class='DiscoverHero'>
+        {{branded-header
+            providerModel=this.providerModel
+            showHelp=true
+            value=(mut this.query)
+            onSearch=(action 'onSearch')
+            searchable=this.searchable
+        }}
+    </HeroOverlay>
+    {{#registries-discover-search
```

Comment:
Since you're changing things in here, could you make all these curly-brace-invocations into angle-bracket-invocations? In this case, it would be `<RegistreisDiscoverSearch` etc etc.